php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -651,8 +652,10 @@ public static void expression(LexerlessGrammarBuilder b) {
   }
 
   public static void declaration(LexerlessGrammarBuilder b) {
+    // FIXME (Linda): USE_CONST_DECLARATION_STATEMENT is never used...
     b.rule(USE_CONST_DECLARATION_STATEMENT).is(USE, CONST, USE_FUNCTION_DECLARATIONS, EOS);
-    b.rule(USE_FUNCTION_DECLARATION_STATEMENT).is(USE, FUNCTION, USE_FUNCTION_DECLARATIONS, EOS); // TODO martin: to check
+    // TODO martin: to check
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)